business/src/test/java/cz/idomatojde/TestObjects.java

```diff
@@ -0,0 +1,70 @@
+package cz.idomatojde;
+
+import cz.idomatojde.entity.Category;
+import cz.idomatojde.entity.Offer;
+import cz.idomatojde.entity.Timetable;
+import cz.idomatojde.entity.TimetableEntry;
+import cz.idomatojde.entity.User;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public final class TestObjects {
+
+    public static User getUser(String username) {
+        User user = new User();
+
+        user.setUsername(username);
+        user.setPassHash("UGFzc3dvcmQ=");
+        user.setPassSalt("U2FsdA==");
+        user.setName("Name");
+        user.setSurname("Surname");
+        user.setPhoneNumber("+420123456789");
+        user.setEmail("pepega@mail.com");
```

Comment:
email should be changed by username because of added unique constraints to user email
f.e. username + "@mail.com"